dev-docs/docs/@excalidraw/excalidraw/api/components/welcome-screen.mdx

```diff
@@ -0,0 +1,141 @@
+# WelcomeScreen
+
+When the canvas is empty, Excalidraw can show a welcome "splash" screen with a logo, a few quick action items, and hints explaining what some of the UI buttons do. Once the user picks a tool, or has created an element on the canvas, the welcome screen will disappear.
+
+You can enable this behavior by rendering a `WelcomeScreen` component like this:
+
+```jsx live
+function ExcalidrawWithMainMenuGroup() {
+  return (
+    <div style={{ height: "500px" }}>
+      <Excalidraw>
+        <WelcomeScreen />
+      </Excalidraw>
+    </div>
+  );
+}
+```
+
+You can also customize the welcome screen by passing children to the `WelcomeScreen` component. See below.
+
+## `<WelcomeScreen/>`
+
+This is the main component. If you render it without any children, we will render the default welcome screen.
+
+You can customize which welcome screen subcomponents are rendered by passing them as children.
+
+The welcome screen consists of two main groups of subcomponents:
+
+1. [WelcomeScreen.Center](#welcomescreencenterlogo).
+2. [WeelcomeScreen.Hints](#welcomescreenhints).
+
+<img
+  src={require("@site/static/img/welcome-screen-overview.png").default}
+  alt="Excalidraw logo: Sketch handrawn like diagrams."
+/>
+
+### `Center`
+
+`<WelcomeScreen.Center/>` subcomponent is the center piece of the welcome screen, containing the `logo`, `heading`, and `menu`. All three subcomponents are optional, and you can render whatever you wish into the center component.
+
+```jsx live
+function ExcalidrawWithWelcomeScreenCenter() {
```

Comment:
I think we should be using `function App()` everywhere to minimize confusion. It should be obvious this is the main component, and the naming doesn't carry any magical meaning.